src/main/java/net/dv8tion/jda/core/entities/Game.java

```diff
@@ -81,15 +140,15 @@ static Game of(String name)
      *
      * @see    #isValidStreamingUrl(String)
```

Comment:
I would also add a note to the url param and maybe reformulate it (the invalid part doesn't make that much sense as user can't specify type anyway)